tools/getbtfhub.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+# clone BTFHub and extract files
+# git clone --depth and -b can't be used together. Let's use a workaround
+# https://stackoverflow.com/questions/31278902/how-to-shallow-clone-a-specific-commit-with-depth-1
+
+git clone --depth 1 https://github.com/aquasecurity/btfhub/ /tmp/btfhub
```

Comment:
URL will need to be updated (see https://github.com/kinvolk/inspektor-gadget/pull/406)

Reply:
Actually not only the URL but much of the script as the tools are now shipped in other repository. Anyway, it's already done.